zcash_primitives/src/transaction/components/orchard.rs

```diff
@@ -213,6 +219,7 @@ pub fn write_v5_bundle<W: Write>(
 
         writer.write_all(&[bundle.flags().to_byte()])?;
         writer.write_all(&bundle.value_balance().to_i64_le_bytes())?;
+        write_bundle_burn(&mut writer, bundle.burn())?;
```

Comment:
Let's inline this function, similar to two lines below.

Reply:
Done.